src/css/blog-post.module.scss

```diff
@@ -1,6 +1,40 @@
 @import "/src/css/colors.scss";
 @import "/src/css/text-styles.scss";
 
+// Social Media
+
+.socialDisappear {
+  transition: transform 0.5s ease-in-out, opacity 0.5s ease-in-out;
+  opacity: 0;
+}
+
+.socialDisappear::after {
+  display: none;
+}
+
+.socialAppear {
+  transition: transform 0.5s ease-in-out, opacity 0.5s ease-in-out;
+  opacity: 100;
+}
+
+.socialBottomContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  width: 89%;
```

Comment:
This is strange, why do you define the width with 89%?

Reply:
That value is to align the SocialBlog component with the blog text, but, if you think it's better, I can change it for a margin.

Reply:
I understand but isn't there another way to do it? How is the blog text aligned?

Reply:
The blog text is aligned with padding. I put padding in the left and right for positioning it.

Reply:
Does this work in every resolution?

Reply:
You are right, in mobile it doesn't work

Reply:
Yes, you have the reason. In addition, in the design in the mobile view, the social component is at the left of the page.